File: Miguel Franco, Oswaldo Romero/Tareas/Menu/Menu.py
from os import system

while True:
    print("Bienvenido, seleccione una opción...")
    print("1-Opción 1")
    print("2-Opción 2")
    print("3-Opción 3")
    print("4-Salir")
    
    opc_ppl = int(input())
    system("cls")
    if opc_ppl == 1:
        while True:
            print("Ha elegido la Opcion 1. Seleccione una opción...")
            print("1-Opción 1.1")
            print("2-Opción 1.2")
            print("3-Opción 1.3")
            print("4-Salir de Opción 1")
            
            opc1 = int(input())
            
            system ("cls")

            if opc1 == 1 or opc1 == 2 or opc1 == 3:
                print("Hola...")
                input("Presione enter para continuar")
                system("cls")

            elif opc1 == 4:
                print("Adiós...")
                input("Presione enter para continuar")
                system("cls")
                break
            else:
                print("Opción inválida...")
                input("Presione enter para continuar")
                system("cls")
            
    elif opc_ppl == 2:
        while True:
            print("Ha elegido la Opcion 2. Seleccione una opción...")
            print("1-Opción 2.1")
            print("2-Opción 2.2")
            print("3-Opción 2.3")
            print("4-Salir de Opción 2")
            
            opc2 = int(input())
            system("cls")

            if opc2 == 1 or opc2 == 2 or opc2 == 3:
                print("Hola...")
                input("Presione enter para continuar")
                system("cls")
            elif opc2 == 4:
                print("Adiós...")
                input("Presione enter para continuar")
                system("cls")
                break
            else:
                print("Opción inválida...")
                input("Presione enter para continuar")
                system("cls")
            

    elif opc_ppl == 3:
        while True:
            print("Ha elegido la Opcion 3. Seleccione una opción...")
            print("1-Opción 3.1")
            print("2-Opción 3.2")
            print("3-Opción 3.3")
            print("4-Salir de Opción 3")
            
            opc3 = int(input())
            system("cls")

            if opc3 == 1 or opc3 == 2 or opc3 == 3:
                print("Hola...")
                input("Presione enter para continuar")
                system("cls")
            elif opc3 == 4:
                print("Adiós...")
                input("Presione enter para continuar")
                system("cls")
                break
            else:
                print("Opción inválida...")
                input("Presione enter para continuar")
                system("cls")
            
    elif opc_ppl == 4:
        print("Adiós...")
        input("Presione enter para continuar")
        system("cls")
        break
    
    else:
        print("Error. Opción inválida...")
        input("Presione enter para continuar")

File: Miguel Franco, Oswaldo Romero/Tareas/Menu/Menu.php
<!DOCTYPE html>
<html>
<head>
    <title>Menu</title>
</head>
<body>

<h2>Bienvenido, seleccione una opción...</h2>
<form method="post">
    <input type="submit" name="opc_ppl" value="Opción 1">
    <input type="submit" name="opc_ppl" value="Opción 2">
    <input type="submit" name="opc_ppl" value="Opción 3">
    <input type="submit" name="opc_ppl" value="Salir">
</form>

<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $opc_ppl = $_POST["opc_ppl"];
    switch ($opc_ppl) {
        case "Opción 1":
            echo "<h2>Ha elegido la Opción 1. Seleccione una Opción...</h2>";
            echo "<form method='post'>
                    <input type='submit' name='opc1' value='Opción 1.1'>
                    <input type='submit' name='opc1' value='Opción 1.2'>
                    <input type='submit' name='opc1' value='Opción 1.3'>
                    <input type='submit' name='opc1' value='Salir de Opción 1'>
                </form>";
            break;
        case "Opción 2":
            echo "<h2>Ha elegido la Opción 2. Seleccione una Opción...</h2>";
            echo "<form method='post'>
                    <input type='submit' name='opc2' value='Opción 2.1'>
                    <input type='submit' name='opc2' value='Opción 2.2'>
                    <input type='submit' name='opc2' value='Opción 2.3'>
                    <input type='submit' name='opc2' value='Salir de Opción 2'>
                </form>";
            break;
        case "Opción 3":
            echo "<h2>Ha elegido la Opción 3. Seleccione una Opción...</h2>";
            echo "<form method='post'>
                    <input type='submit' name='opc3' value='Opción 3.1'>
                    <input type='submit' name='opc3' value='Opción 3.2'>
                    <input type='submit' name='opc3' value='Opción 3.3'>
                    <input type='submit' name='opc3' value='Salir de Opción 3'>
                </form>";
            break;
        case "Salir":
            echo "<h2>Adiós...</h2>";
            break;
        default:
            echo "<h2>Error. Opción Inválida...</h2>";
            break;
    }
}
?>

</body>
</html>
